modules/classic/build.sh

```diff
@@ -72,6 +72,8 @@ if [[ $PROTO == 'https://' ]]; then
   mysql -h "$mysqlHost" -u root -p"${MYSQL_ROOT_PASSWORD}" "$SHOPWARE_PROJECT" -e "UPDATE s_core_shops SET secure = 1 WHERE main_id IS NULL"
 fi
 
+$0 snap "$SHOPWARE_PROJECT"
```

Comment:
I am not sure if this needs to be done soo early. You don't need snippets and so?

Reply:
Has been fixed